src/ui.tsx

```diff
@@ -1,39 +1,101 @@
-import * as React from "react";
+import React, { useState, useEffect } from "react";
 import * as ReactDOM from "react-dom/client";
+import { CopyToClipboardButton } from 'react-clipboard-button';
+import JSONPretty from 'react-json-pretty';
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const JSONPrettyMon = require('react-json-pretty/themes/monikai.css');
 import "./ui.css";
 
-declare function require(path: string): string;
 
 function App() {
- const [output, setOutput] = React.useState(0);
+  // Set up the state for the output
+  const [output, setOutput] = useState<string>('');
+  const [readyToCopy, setReadyToCopy] = useState(false);
```

Comment:
One final tiny nit would be to add the type declaration for useState here as well for consistency (<boolean>).